documentation/guide_to_issues_on_github.md

```diff
@@ -0,0 +1,107 @@
+# Guide to GitHub Issues
+
+## Description
+To ensure that our team handles GitHub issues correctly and consistently. This guide will serve as a quick reference for all team members, new and experienced, by outlining best practices for creating, addressing, and resolving issues. Our intention is to streamline our workflow, reduce misunderstandings, and ensure that all issues are addressed swiftly and without the need for follow-up questions. If you are new to GitHub and issues, start [here](https://docs.github.com/en/issues).
+
+## Objectives
+**Consistency**: Establish a common procedure for creating, discussing, and resolving issues in all repositories.
+
+**Efficiency**: Ensure that all relevant information is included in issues from the outset, and that they are properly assigned, prioritized, and monitored.
+
+**Clarity**: Provide clear, actionable guidelines for communication within issues, including how to ask for help, provide updates, and close issues when resolved.
+
+## Templates
+
+For issues related to [bugfixes](https://github.com/prio-data/views_pipeline/blob/github_issue_guide/documentation/github_bugfix_issue_template.md) and [feature requests](https://github.com/prio-data/views_pipeline/blob/github_issue_guide/documentation/github_feature_request_template.md).
+
+## Guide Content
+### Creating an Issue
+1. **Writing a Clear and Concise Issue Title**
+    * Use descriptive and specific titles.
+    * Avoid vague terms like “bug” or “issue.”
+    
+2. **Essential Details to Include in the Issue Description**
+    * Steps to Reproduce: Detailed steps to replicate the issue.
+    * Expected Behavior: What should happen.
+    * Actual Behavior: What actually happens.
+    * Screenshots/Logs: Attach relevant screenshots or logs.
+
+3. **Guidelines for Labeling Issues**
+    * Use predefined labels such as `bug`, `documentation`, `help wanted`, `improvement`. See [label guide](https://github.com/prio-data/views_pipeline/labels) for more information.
+    * Ensure labels are applied consistently.
+
+4. **Assigning Issues**
+    * Assign issues to the appropriate team members based on expertise.
+    * Use the Assignees feature in GitHub.
```

Comment:
I think we can add that usually 2 people should be assigned - and for very substantial changes to working solutions, more might be needed.

Reply:
Done